srcpkgs/mkvtoolnix/template

```diff
@@ -12,10 +12,10 @@ makedepends="boost-devel cmark-devel file-devel json-c++
 short_desc="Create, alter and inspect Matroska videos"
 maintainer="Matthias von Faber <mvf@gmx.eu>"
 license="GPL-2.0-only"
-homepage="https://www.bunkus.org/videotools/mkvtoolnix/"
-changelog="https://mkvtoolnix.download/doc/NEWS.md"
-distfiles="https://www.bunkus.org/videotools/${pkgname}/sources/${pkgname}-${version}.tar.xz"
-checksum=7cdd6ad9144324162604172b7e96cb13cdbb7913e07c4ad7633344aac20e4103
+homepage="https://mkvtoolnix.download"
+changelog="${homepage}/doc/NEWS.md"
+distfiles="${homepage}/sources/${pkgname}-${version}.tar.xz"
```

Comment:
Please don't use `${homepage}` here but the URL because the fields are not always expanded, e.g. https://alpha.de.repo.voidlinux.org/void-updates/void-updates.txt .

Reply:
Fixed, thanks for the hint! Wasn't sure myself about this, but there are ~500 other packages doing this, so I assumed it was okay.

Reply:
IMHO only packages where the leading part of the URL is very long should do this. There is no written rule but just common sense. Mentally parsing an expression with variables also takes its time and brain capacity :)